firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -84,6 +92,14 @@ SCENARIO( "Turn_Debugging_Publishes_On_Off", "[toggle debug flag]" ) {
                 REQUIRE( returnVal == -1 );
             }
         }
+
+        WHEN("The function is called with 'e'") {
+            int returnVal = toggle_debugging_publishes("e");   
+
+            THEN("The function should return whether debugging publishes are turned on, in this case, true") {
+                REQUIRE(returnVal == true);
```

Comment:
This should test it against the integer `1`, right?

Reply:
I changed the true to a 1 in my latest commit